scss/_off.scss

```diff
@@ -1451,3 +1451,50 @@ a.panel_title {
     background-color: scale-color($button-bg-color, $lightness: -$button-function-factor / 2);
     color: #fff;
 }
+// organization members table in producers platform
```

Comment:
We can do it in a separate PR, but it would be a good thing to see if we can make the CSS code that we have for packagings generic, so that it can be applied to any table, instead of having similar separate CSS code for each table.

Reply:
Yes..It would be good to make the CSS generic for these tables👍...would open a separate PR for CSS things..will do it asap!